create-job-ui/src/pages/CreateJobPage.css

```diff
@@ -0,0 +1,14 @@
+.create-job-page{
+    padding: 10;
+    border: 5px solid black;
+    padding: 50px 50px;
+    align-items: center;
+    text-align: center;
+    background-color: antiquewhite;
+    width:50vw;
+
+}
+.mainContainer{
```

Comment:
@asharMehmood2211004KHICND try to use consistent casing, hyphens are standard in most CSS e.g. `main-container` rather than `mainContainer`